assets/src/features/leaving-page/icons/🚀.tsx

```diff
@@ -0,0 +1,69 @@
+        fill="#001A72"
+      />
+      <path
+        d="M126.064 43.0326C115.502 44.6307 78.0055 53.7495 77.9899 102.196L102.327 116.155C144.135 91.6783 133.657 55.094 129.867 45.2137C129.591 44.4652 129.059 43.8368 128.364 43.4381C127.669 43.0395 126.855 42.896 126.064 43.0326Z"
+        fill="#D2CFFD"
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M125.86 41.7573C126.946 41.5732 128.062 41.7718 129.015 42.3184C129.969 42.866 130.701 43.7287 131.083 44.7573C133.017 49.8019 136.661 61.6585 134.479 75.4089C132.285 89.2373 124.219 104.836 102.984 117.268C102.58 117.504 102.081 117.507 101.677 117.275L77.3391 103.316C76.9381 103.086 76.6913 102.661 76.6914 102.2C76.6994 77.5902 86.2518 62.8246 97.2614 54.0393C108.203 45.3084 120.464 42.5751 125.86 41.7573ZM127.713 44.5583C127.28 44.3101 126.774 44.2211 126.283 44.3059C126.277 44.307 126.271 44.308 126.265 44.3089C126.262 44.3094 126.259 44.3099 126.256 44.3103C121.083 45.093 109.325 47.7207 98.8827 56.0535C88.6081 64.2523 79.4885 78.0638 79.2918 101.45L102.321 114.658C122.404 102.681 129.865 87.9276 131.914 75.013C133.996 61.8884 130.51 50.5174 128.654 45.6777L128.648 45.6624C128.476 45.1975 128.146 44.8065 127.713 44.5583Z"
+        fill="#001A72"
+      />
+      <path
+        d="M108.602 77.443C111.122 78.8885 114.345 78.0307 115.8 75.5271C117.255 73.0234 116.391 69.822 113.871 68.3765C111.351 66.9311 108.128 67.7889 106.673 70.2925C105.218 72.7962 106.081 75.9976 108.602 77.443Z"
+        fill="#001A72"
+      />
+      <path
+        d="M77.2557 130.05C77.4855 129.652 78.1741 129.62 78.7937 129.977C79.4132 130.335 79.7291 130.948 79.4993 131.346L74.114 140.673C73.8842 141.071 73.1956 141.104 72.5761 140.746C71.9565 140.389 71.6406 139.776 71.8704 139.378L77.2557 130.05Z"
+        fill="#001A72"
+      />
+      <path
+        d="M69.5041 125.64C69.7494 125.215 70.4505 125.161 71.07 125.518C71.6896 125.876 71.993 126.51 71.7477 126.935L61.7551 144.243C61.5098 144.668 60.8088 144.722 60.1892 144.364C59.5697 144.007 59.2662 143.372 59.5115 142.948L69.5041 125.64Z"
+        fill="#001A72"
+      />
+      {/* <path d="M84.9166 134.618C85.1808 134.16 85.8973 134.079 86.5168 134.437L6.5168 134.437C87.1364 134.794 87.4244 135.455 87.1602 135.913L71.5367 162.974C71.2725 163.431 70.5561 163.512 69.9365 163.155L69.9365 163.155C69.317 162.797 69.0289 162.136 69.2931 161.678L84.9166 134.618Z" fill="#001A72"/> */}
```

Comment:
I've talked with Alicja. We've established it is a problem with Figma because the problem is easily replicable on every device with the given rocket `svg`. The `path` element contains nan each and every time.

She'll ping Balbina and we will see what can be done.

Reply:
Done 🚀 Shoutout to Balbina